recipes/triqs_cthyb/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+mkdir build
+cd build
+
+# Openmpi Specific environment setup - Cf. https://github.com/conda-forge/libnetcdf-feedstock/pull/80
+export OMPI_MCA_btl=self,tcp
+export OMPI_MCA_plm=isolated
+export OMPI_MCA_rmaps_base_oversubscribe=yes
+export OMPI_MCA_btl_vader_single_copy_mechanism=none
+mpiexec="mpiexec --allow-run-as-root"
+
+source $PREFIX/share/triqsvars.sh
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
This should build a release

Reply:
We do in fact build in Release mode by default. See associated [CMakeLists.txt](https://github.com/TRIQS/cthyb/blob/fd314234082503ad7907944817be44410d67a84a/CMakeLists.txt#L56-L60)

```cmake
# Default to Release build type
if(NOT CMAKE_BUILD_TYPE)
  set(CMAKE_BUILD_TYPE Release CACHE STRING "Type of build" FORCE)
endif()
message(STATUS "-------- BUILD-TYPE: ${CMAKE_BUILD_TYPE} --------")
```

Reply:
I anyways added the build type explicitly